docs/sdk-and-tools/erdjs/erdjs-cookbook.md

```diff
@@ -0,0 +1,573 @@
+---
+id: erdjs-cookbook
+title: Cookbook
+---
+
+This page will guide you through the process of handling common tasks using **erdjs**.
+
+:::important
+This cookbook makes use of `erdjs 10`.
```

Comment:
please reference the migration guide here

Reply:
Added.